src/qml/QFieldSettings.qml

```diff
@@ -33,41 +33,41 @@ Page {
   ListModel {
       id: settingsModel
       ListElement {
-          title: QT_TR_NOOP( "Show scale bar" )
```

Comment:
I don't recall why I switched to QT_TR_NOOP, but I had a good reason to do so...
see maybe an alternative: #1092